src/pages/index.tsx

```diff
@@ -40,7 +41,68 @@ const Hero = () => {
   );
 };
 
-// const Services = () => {}
+const Services = () => {
+  const Services = [
+    {
+      id: 1,
+      icon: "web.svg",
+      borderColor: "border-orange/20",
+      title: "Websites",
+      description:
+        "Fast. Responsive. Accessible.\nWe specialize in building high-performing websites without leaving any users behind. ",
+    },
+    {
+      id: 2,
+      icon: "mobile.svg",
+      borderColor: "border-green/20",
+      title: "Mobile Apps",
+      description:
+        "We build cross-platform mobile apps. Lorem ipsum dolor sit amet consectetur adipiscing elit.",
+    },
+    {
+      id: 3,
+      icon: "design.svg",
+      borderColor: "border-red/20",
+      title: "UI/UX Design",
+      description:
+        "First impressions matter. We know how to craft unique and impressive digital experiences.",
+    },
+  ];
+
+  return (
+    <div className="my-10 bg-sand">
+      <Container>
```

Comment:
I replaced `<div className="container">` with `<Container>`. Right now, they are identical, but I want to move away from Tailwind's container because I don't love it.